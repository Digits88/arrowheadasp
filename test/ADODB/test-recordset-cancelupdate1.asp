<!--#include file="../config/ADODB.inc"-->
Here is the test of RecordSet CancelUpdate.<br>
<%
    Dim CN, RS

    Set CN = Server.CreateObject("ADODB.Connection")
    CN.Open Connect_URL, Connect_Username, Connect_Password
    Set RS = Server.CreateObject("ADODB.RecordSet")
    RS.Open "SELECT StringField, IntField FROM Test WHERE IntField = 99 ORDER BY IntField", CN, 2, 2

    RS.AddNew
    RS.Fields(0) = "New String Field"
    RS.Fields(1) = 99
    RS.CancelUpdate

    RS.Close

    Set RS = CN.Execute("SELECT StringField, IntField FROM Test ORDER BY IntField")
    Call Output_Table(RS)
    RS.Close

    CN.Close

%>

The End.
