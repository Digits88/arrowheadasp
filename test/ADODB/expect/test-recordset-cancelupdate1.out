
Here is the test of RecordSet CancelUpdate.<br>
Value: First Row / 1<br>
        Value: Second Row / 2<br>
        Value: Third Row / 3<br>
        Value: Fourth Row / 4<br>
        Value: Fifth Row / 5<br>
        Value: Sixth Row / 6<br>
        Value: Seventh Row / 7<br>
        Value: Eigth Row / 8<br>
        Value: Ninth Row / 9<br>
        Value: Tenth Row / 10<br>
        Value: Eleventh Row / 11<br>
        Value: Twelfth Row / 12<br>
        Value: Thirteenth Row / 13<br>
        Value: Fourteenth Row / 14<br>
        Value: Fifteenth Row / 15<br>
        

The End.
